kotest-framework/kotest-framework-api/src/commonMain/kotlin/io/kotest/core/filter/Filter.kt

```diff
@@ -1,3 +1,6 @@
 package io.kotest.core.filter
 
+
 interface Filter
+
+annotation class SpecClassExecutionFilter(val filter: String)
```

Comment:
Or we can use not the String - something from new Interface (and the users can implement this for their needs as they want)